modules/integration/tests-integration/tests-backend/src/test/resources/org/wso2/identity/integration/test/rest/api/server/idp/v1/add-idp-oidc-standard-based.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "OIDC IdP",
```

Comment:
Check whether there are any existing json similar to this, so we can use it.

Reply:
There seems no any similar files we could utilize for this test case.